### DISCUSSION TITLE
job position

### DISCUSSION THREAD

**Discussion by jamiel3431** [arkypita/LaserGRBL#2460] [Q&A]
how can i either move the job to the right side of zero or move zero to the other side

**Comment by Tribo-2dot0** [comment]
Insira coordenadas negativas em "Posição"...
![Captura de tela 2024-12-17 002331](https://github.com/user-attachments/assets/35cf5a60-9cdb-4f52-9f91-393d29807a42)

**Reply by jamiel3431** [reply]
it does not give me that option all it gives me is s min and smax

**Reply by Tribo-2dot0** [reply]
então você deve estar abrindo um arquivo .svg
o svg terás que posiciona-lo dentro de um editor svg... abra-o em Inkscape, selecione tudo e ajuste a página à seleção (menu Editar > Ajustar Página à Seleção), arraste o projeto (seleção) para a esquerda e/ou para baixo.
![Captura de tela 2024-12-17 124959](https://github.com/user-attachments/assets/8bf8e032-8dc4-4420-8302-23a0a1048d22)
![Captura de tela 2024-12-17 125128](https://github.com/user-attachments/assets/b947612f-d32a-462e-941e-4bc711297be3)
